web/src/test/java/com/replica/RouterServerTest.java

```diff
@@ -398,6 +433,9 @@ private void checkTransitQuery(RouterOuterClass.PtRouteReply response,
         for (int i = 0; i < path.getLegsList().size(); i++) {
             assertEquals(expectedStableEdgeIdCount.get(i), path.getLegsList().get(i).getStableEdgeIdsCount());
         }
+
+        // Check number of transfers is correct
```

Comment:
Feeling relieved that this new check passes 😌 cc @rregue this `transfers` field in PT response proto objects should be a reliable way to track transfer counts vs. doing some calculation on the client side (although this check shows that counting the number of `TRANSFER` walking legs should work too for an accurate count)

Reply:
So then, this property in the Route object will be accurate https://github.com/replicahq/model/blob/58659de6ed3ea644b2b2cd3fe29aa585e326406c/mapservice/api/route.py#L156. I see it was only used in the RouterPerformance for now, but we have other places where we have n_connections, that we could just replace and use this property of the Route.

Reply:
Correct, after this change is integrated into TN CDP this property will now be accurate! Maybe in the model repo PR where I integrate this graphhopper change, I can update the `n_connections` occurrences to rely on this property vs. calculating transfers another way